languages/bevy_mod_scripting_lua/Cargo.toml

```diff
@@ -11,6 +11,9 @@ keywords = ["bevy", "gamedev", "scripting", "rhai"]
 categories = ["game-development"]
 readme="readme.md"
 
+[package.metadata."docs.rs"]
+features = ["lua54","lua_script_api","teal"]
```

Comment:
I believe only the lua54 feature should be included as, the others are not from this crate:
```toml
# enables loading possibly unsafe lua modules by lua scripts
unsafe_lua_modules = []

# enable teal utilities
teal = []

lua51 = ["tealr/mlua_lua51"]
lua52 = ["tealr/mlua_lua52"]
lua53 = ["tealr/mlua_lua53"]
lua54 = ["tealr/mlua_lua54"]
luajit = ["tealr/mlua_luajit"]
luajit52 = ["tealr/mlua_luajit52"]
mlua_serialize = ["tealr/mlua_serialize"]
mlua_macros = ["tealr/mlua_macros"]
mlua_async = ["tealr/mlua_async"]
```